src/components/Button/Button.js

```diff
@@ -24,13 +24,13 @@ Btn.propTypes = {
   disabled: PropTypes.bool,
   children: PropTypes.node,
   onPress: PropTypes.func,
-  styles: PropTypes.object,
+  style: PropTypes.object,
   textStyling: PropTypes.object
 };
 
 const styles = StyleSheet.create({
   button: {
-    backgroundColor: colors.signed_primary,
+    backgroundColor: COLORS.signed_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `style` prop type has been corrected and the background color for the button has been updated to use `COLORS.signed_primary` from the `theme` module instead of `colors.signed_primary` from the `colors` module.
```diff
-  styles: PropTypes.object,
+  style: PropTypes.object,
-  backgroundColor: colors.signed_primary,
+  backgroundColor: COLORS.signed_primary,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->